test/e2e_node/container_manager_test.go

```diff
@@ -150,6 +150,7 @@ var _ = framework.KubeDescribe("Container Manager Misc [Serial]", func() {
 				})
 				// Log the running containers here to help debugging. Use `docker ps`
 				// directly for now because the test is already docker specific.
+				// TODO(random-liu): Use `crictl ps` and `crictl sandboxes` instead.
```

Comment:
Why can't you use the CRIClient you add in this PR to do this?

Also, without changing this, wouldn't the test still fail for non-docker runtimes? How do we currently skip the test for rkt and and non-docker CRI runtimes?

Reply:
@yujuhong If the test doesn't fail, this line won't be run.
I'll change this to use CRIClient to dump debug information.

Reply:
Done.